coral/src/app/layout/main-navigation/MainNavigationLink.tsx

```diff
@@ -1,9 +1,9 @@
 import { Box, Flexbox, Icon } from "@aivenio/design-system";
 import data from "@aivenio/design-system/dist/src/icons/console";
-import classes from "src/app/layout/MainNavigationLink.module.css";
+import classes from "src/app/layout/main-navigation/MainNavigationLink.module.css";
```

Comment:
Question: I notice we used the full path rather than the relative path (like `"./MainNavigationLink.module.css"`) for module imports. Is it a stylistic choice or is it a technical requirement?

Reply:
That was a stylistic choice we made :D Mostly to not have a lot of changed lines when doing refactoring and moving stuff around. If we are more stable in how our structure in general should look like, we also could create different aliases for different areas. 🤔  

The imports that way are also a bit more readable - of course they are longer, but it's very clear where an import is coming from.  (super small benefit, because usually one does not really _read_ import paths I would say).